package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "last-train-to-nodeville",
+  "version": "1.0.0",
+  "description": "## Instructions",
+  "main": "index.js",
+  "dependencies": {
+    "chai": "^4.2.0",
+    "mocha": "^6.2.0"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "mocha tests.js"
```

Comment:
Can you confirm what you get as the output of running npm `run test`?
I would expect running ` mocha` to result in 
`$ mocha
bash: mocha: command not found`

See slide 52 for a hint in the scripts for a scrip that is closer to what I would expect this value to be here.

Reply:
![image](https://user-images.githubusercontent.com/32113320/65366936-638bf600-dbf8-11e9-8707-4384ed9c3010.png)

Reply:
this is before making any changes to the test script
![image](https://user-images.githubusercontent.com/32113320/65366979-e9a83c80-dbf8-11e9-8c87-8e2c21cf73f3.png)

Reply:
![image](https://user-images.githubusercontent.com/32113320/65367244-6c7ec680-dbfc-11e9-812f-3821b890f973.png)
![image](https://user-images.githubusercontent.com/32113320/65367252-7ef90000-dbfc-11e9-83d6-3752b5c0ac19.png)
This is after the script was corrected

Reply:
![image](https://user-images.githubusercontent.com/32113320/65367474-3d6a5400-dc00-11e9-89bb-8e9242e93df2.png)
![image](https://user-images.githubusercontent.com/32113320/65367477-49eeac80-dc00-11e9-9bed-164329608539.png)
This is after the recommended changes were made